crates/evm/evm/src/executors/invariant/mod.rs

```diff
@@ -97,11 +97,86 @@ sol! {
     }
 }
 
+/// Contains invariant test data.
+pub struct InvariantTest {
+    // Fuzz state of invariant test.
+    pub fuzz_state: EvmFuzzState,
+    // Contracts fuzzed by the invariant test.
+    pub targeted_contracts: FuzzRunIdentifiedContracts,
+    // Consumed gas and calldata of every successful fuzz call.
+    pub fuzz_cases: RefCell<Vec<FuzzedCases>>,
```

Comment:
we could remove all the inner ref cells and only have one that wraps the entire struct

Reply:
please check https://github.com/foundry-rs/foundry/pull/8236/commits/ae597a95d2310f0b134f5d56e42eb888b0650731